src/test/tpdifficulty/tp/AiModtpDifficulty.java

```diff
@@ -94,14 +98,35 @@ public void CalculateAll(ArrayList<HitObject> hitObjects, float circleSize) {
         // Fill our custom tpHitObject class, that carries additional information
         tpHitObjects = new ArrayList<tpHitObject>(hitObjects.size());
         float CircleRadius = (PLAYFIELD_WIDTH / 16.0f) * (1.0f - 0.7f * (circleSize - 5.0f) / 5.0f);
-
+        //float CircleRadius = (PLAYFIELD_WIDTH / 16.0f) * (1.0f - 0.7f * (circleSize - 5.0f) / 5.0f);
```

Comment:
Should be removed, this is an unnecessary comment.

Reply:
fixed